src/Document/ProductBlock.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Lakion\SyliusCmsBundle\Document;
+
+use Sylius\Component\Core\Model\ProductInterface;
+
+class ProductBlock extends CustomBlock
+{
+    /**
+     * @var ProductInterface
+     */
+    private $product;
```

Comment:
Isn't product block something entitylike? Shoulgn't thiese fields be protected instead of private?